extensions/Lily/Data.js

```diff
@@ -527,13 +527,11 @@
     orderIs(args, util) {
       const list = getVarObjectFromName(args.LIST, util, "list");
       if (!list) return false;
-      const defaultList = cloneObj(list.value);
-      const orderedList = cloneObj(list.value).sort();
-      if (args.ORDER === "ascending") {
-        return arraysEqual(defaultList, orderedList);
-      }
-      if (args.ORDER === "descending") {
-        return arraysEqual(defaultList.reverse(), orderedList);
+
+      for (let i = 0; i < list.value.length - 1; i++) {
+        const compare = Scratch.Cast.compare(list.value[i + 1], list.value[i]);
+        if (compare < 0 && args.ORDER === "descending") return true;
```

Comment:
![image](https://github.com/TurboWarp/extensions/assets/33787854/8c26c6e9-d9d0-49f5-9122-e44d1de8821e)

Reply:
oh for griff's sake.